packages/ssx-react/src/ssx.tsx

```diff
@@ -2,12 +2,22 @@ import { useContext, createContext, useState, useEffect, ReactNode } from 'react
 import { SSX, SSXClientConfig } from "@spruceid/ssx";
 import { useSigner } from 'wagmi';
 
+/** Interface for SSX Web3 Provider. */
+export interface SSXWeb3Provider {
+  /** web3 Provider. */
+  provider: any;
+  /** web3 Provider Loaded. */
+  providerLoaded: boolean;
+}
+
 /** Props for SSX Provider. */
 export interface SSXProviderProps {
   /** Optional SSX configuration, used for instantiating an SSX Instance. */
   ssxConfig?: SSXClientConfig;
   /** Provider child nodes, for rendering. */
   children: ReactNode;
+  /** Optional SSX Signer. Will . */
+  web3Provider?: SSXWeb3Provider;
```

Comment:
@w4ll3 @Juliano1612 Should I move these to ssx-core? They are really only used in ssx-react

Reply:
I suppose it's fine to keep it then.

Reply:
Agree with @w4ll3